webcompat/templates/addon-links.html

```diff
@@ -42,7 +42,7 @@
     <span class="wc-Navbar-linkDownload-label-word">Download our </span> Opera Extension
  </span>
 </a>
-{%- elif browser == 'safari' -%}
+{%- elif browser == 'safari WHEN IT IS READY' -%}
```

Comment:
This is clever. :)

Even thought it achieves what we want, I think were better off if we're not shipping "dead" code. 

Can you remove the entire `elif` clause, and the addons directory as well? When we're ready to add it back, its not hard to dig up what we had from the git history.

Thanks!